cmd/nerdctl/pull.go

```diff
@@ -92,6 +102,59 @@ func pullAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if isVerify, err := cmd.Flags().GetString("verify"); err == nil && isVerify == "cosign" {
+
+		ref, err := name.ParseReference(rawRef)
+		if err != nil {
+			logrus.Errorf("Unable to parse reference: %v\n", err)
+		}
+
+		digest, err := remote.ResolveDigest(ref)
```

Comment:
Ideally we should rather extend `nerdctl/pkg/imgutil` to support resolving digest, without relying on `github.com/sigstore/cosign/pkg/oci/remote`. Can be revisited later.

Reply:
done @AkihiroSuda